curvefs/src/common/s3util.h

```diff
@@ -32,11 +32,16 @@ namespace s3util {
 inline std::string GenObjName(uint64_t chunkid, uint64_t index,
                               uint64_t compaction, uint64_t fsid,
                               uint64_t inodeid) {
-    return std::to_string(fsid) + "_" + std::to_string(inodeid) + "_" +
+    return std::to_string(fsid) + "/" +
+           std::to_string(inodeid/1000/1000) + "/" +
+           std::to_string(inodeid/1000) + "/" +
```

Comment:
We need a comment that exemplifies what each field means.

Reply:
Why do you need inodeid/1000 and inodeid/1000/1000?

Reply:
> Why do you need inodeid/1000 and inodeid/1000/1000?
for  multi directory and  s3 prefix，but its better to  replace it with hash or %